server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -165,7 +165,7 @@ const accountMutations = {
       },
       action: {
         type: new GraphQLNonNull(
-          new GraphQLEnumType({ name: 'AccountFreezeAction', values: { FREEZE: {}, UNFREEZE: {} } }),
+          new GraphQLEnumType({ name: 'AccountFreezeAction', values: { FREEZE: {}, UNFREEZE: {}, FREEZEKEEPCONTRIBUTIONS: {} } }),
```

Comment:
This `FREEZEKEEPCONTRIBUTIONS` is a bit aggressive and won't play well with the rest of the issue that we'll later want to implement: being able to select the features you want to freeze, We should keep FREEZE/UNFREEZE and add a new param for this